app/src/main/java/org/sopt/and/screen/LogInScreen.kt

```diff
@@ -0,0 +1,303 @@
+                                        )
+                                    when (result) {
+                                        SnackbarResult.ActionPerformed -> {
+                                            Toast.makeText(
+                                                context,
+                                                "id :$signUpId\npassword :$signUpPassword",
+                                                Toast.LENGTH_SHORT
+                                            ).show()
+                                        }
+                                        SnackbarResult.Dismissed -> {
+                                        }
+                                    }
+                                }
+                            } else {
+                                scope.launch {
+                                    Toast.makeText(context, "로그인 하였습니다", Toast.LENGTH_SHORT).show()
+                                }
+                                val intent2 = Intent(context, MyActivity::class.java).apply {
+                                    this.putExtra("profileid", id)
+                                    flags =
+                                        Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_NEW_TASK
+                                }
+                                context.startActivity(intent2)
+                            }
+                        } else {
+                            scope.launch {
+                                val result = snackbarHostState
+                                    .showSnackbar(
+                                        message = "회원가입을 진행해주세요",
+                                        actionLabel = "회원가입하기",
+                                        duration = SnackbarDuration.Short
+                                    )
+                                when (result) {
+                                    SnackbarResult.ActionPerformed -> {
+                                        val intent = Intent(context, SignUpActivity::class.java)
+                                        launcher.launch(intent)
+                                    }
+
+                                    SnackbarResult.Dismissed -> {
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(50.dp)
+                ) {
+                    Text("로그인")
+                }
+                TextButton(
+                    onClick = {
+                        val intent = Intent(context, SignUpActivity::class.java)
+                        launcher.launch(intent)
+                    }
+                ) {
+                    Text("회원가입하기")
+                }
+            }
+
+            Spacer(Modifier.padding(10.dp))
```

Comment:
```suggestion
            Spacer(Modifier.heigth(10.dp))
```

지금그대로 주시면 높이는 20dp 입니다.

Reply:
와.. 감사합니다 정말 완전 놓치고 있었습니다